expr/http_response.go

```diff
@@ -139,6 +139,17 @@ func (r *HTTPResponseExpr) Validate(e *HTTPEndpointExpr) *eval.ValidationErrors
 	if isrt {
 		inview = " all views in"
 	}
+
+	// text/html can only encode strings so make sure that
```

Comment:
Please finish comment :) We should also allow for `Bytes` because that's what the Go templates produce.